libs/sdk-common/src/lnurl/specs/auth.rs

```diff
@@ -1,39 +1,50 @@
 use std::str::FromStr;
 
-use bitcoin::hashes::{hex::ToHex, sha256, Hash, HashEngine, Hmac, HmacEngine};
-use bitcoin::secp256k1::{Message, Secp256k1};
-use bitcoin::util::bip32::{ChildNumber, ExtendedPrivKey};
-use bitcoin::KeyPair;
+use bitcoin::hashes::hex::ToHex;
+use bitcoin::util::bip32::ChildNumber;
 use reqwest::Url;
 
 use crate::prelude::*;
 
+pub trait LnurAuthSigner {
```

Comment:
Sorry, just spotted:
```suggestion
pub trait LnurlAuthSigner {
```

Reply:
Thank you. Done.